app/src/main/AndroidManifest.xml

```diff
@@ -14,93 +14,108 @@
+            android:theme="@style/Theme.H3LP">
         <activity
-            android:name=".SettingsActivity"
+                android:name=".professional.ProfessionalTypeSelection"
+                android:exported="false" />
+        <activity
+            android:name=".forum.ForumCategoriesActivity"
             android:exported="false" />
         <activity
-            android:name=".HelpPageActivity"
-            android:exported="true" />
+            android:name=".forum.ForumAnswersActivity"
+            android:exported="false" />
         <activity
-            android:name=".messaging.RecentMessagesActivity"
+            android:name=".forum.NewPostActivity"
             android:exported="true" />
         <activity
-            android:name=".MainPageActivity"
+            android:name=".forum.ForumPostsActivity"
             android:exported="false" />
         <activity
-            android:name=".professional.VerificationActivity"
+            android:name=".professional.ProProfileActivity"
             android:exported="false" />
         <activity
-            android:name=".professional.ProMainActivity"
+            android:name=".SettingsActivity"
             android:exported="false" />
         <activity
-            android:name=".MySkillsActivity"
-            android:exported="false" />
+                android:name=".HelpPageActivity"
+                android:exported="true" />
         <activity
-            android:name=".firstaid.AsthmaActivity"
-            android:exported="false" />
+                android:name=".messaging.RecentMessagesActivity"
+                android:exported="true" />
         <activity
-            android:name=".firstaid.AedActivity"
-            android:exported="false" />
+                android:name=".MainPageActivity"
+                android:exported="false" />
         <activity
-            android:name=".firstaid.HeartAttackActivity"
-            android:exported="false" />
+                android:name=".professional.VerificationActivity"
+                android:exported="false" />
         <activity
-            android:name=".firstaid.AllergyActivity"
-            android:exported="true" />
+                android:name=".professional.ProMainActivity"
+                android:exported="false" />
         <activity
-            android:name=".FirstAidActivity"
-            android:exported="false" />
+                android:name=".MySkillsActivity"
+                android:exported="false" />
+        <activity
+                android:name=".firstaid.AsthmaActivity"
+                android:exported="false" />
+        <activity
+                android:name=".firstaid.AedActivity"
+                android:exported="false" />
+        <activity
+                android:name=".firstaid.HeartAttackActivity"
+                android:exported="false" />
+        <activity
+                android:name=".firstaid.AllergyActivity"
+                android:exported="true" />
```

Comment:
is there a particular reason this activity is exported?

Reply:
No ! Just forgotten I guess